.gitignore

```diff
@@ -194,3 +194,6 @@ FakesAssemblies/
 
 # Visual Studio 6 workspace options file
 *.opt
+
+# Ignore .idea/ Configuration files
+.idea/
```

Comment:
You're very right!! Thank you :)

On Sat, May 30, 2015 at 5:02 PM, Emmanuel San Miguel <
notifications@github.com> wrote:

> In .gitignore
> <https://github.com/rroa/marvel-csharp/pull/16#discussion_r31384455>:
>
> > @@ -194,3 +194,6 @@ FakesAssemblies/
> >
> >  # Visual Studio 6 workspace options file
> >  *.opt
> > +
> > +# Ignore .idea/ Configuration files
> > +.idea/
>
> @oscarbatlle <https://github.com/oscarbatlle> This is fine, but I would
> consider adding this to your global gitignore
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rroa/marvel-csharp/pull/16/files#r31384455>.
>